packages/suite-web/e2e/tests/settings/t2t1-device-settings.test.ts

```diff
@@ -83,7 +83,7 @@ describe('T2T1 - Device settings', () => {
         cy.passThroughInitialRun();
 
         cy.log('Try to change device homescreen');
-        cy.getTestElement('@settings/device/homescreen').scrollIntoView();
+        cy.getTestElement('@device-settings/homescreen').scrollIntoView();
```

Comment:
you can import anchor constants here and use it

Reply:
239dd1703997060338d34ea7ae49b149e94a033c